crates/miden-objects/src/account/builder/mod.rs

```diff
@@ -100,15 +120,22 @@ impl AccountBuilder {
         #[cfg(all(not(feature = "testing"), not(test)))]
         let vault = AssetVault::default();
 
-        let (code, storage) =
-            Account::initialize_from_components(self.account_type, &self.components).map_err(
-                |err| {
-                    AccountError::BuildError(
-                        "account components failed to build".into(),
-                        Some(Box::new(err)),
-                    )
-                },
-            )?;
+        let auth_component = self
+            .auth_component
+            .clone()
+            .ok_or(AccountError::BuildError("auth component must be set".into(), None))?;
```

Comment:
Nit: Can you add `with_auth_component` to this section in the builder docs: "The methods that are required to be called are"?

Reply:
Done in [4c6219d](https://github.com/0xMiden/miden-base/pull/1480/commits/4c6219d7128bacbf41bc07709b89507e21dde128)